src/__mocks__/index.tsx

```diff
@@ -0,0 +1,32 @@
+const NOOP = () => {
+  // noop
+};
+
+export enum VideoQuality {
```

Comment:
is there some way to import this from the real `index.tsx`? we're going to forget to change this here

Reply:
Added import and export for all of the enums